### ISSUE TITLE
refetchEvents of a single source only

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#1328]
Originally reported on Google Code with ID 1061
```
I'd like to see an argument added to the refetchEvents method which would allow me to
only re-fetch events from a specific source instead of all. All the other events should
be kept in local cache.

Please find attached a patch to add this functionality.

Thanks a lot for this great piece of software!
```

Reported by `roundcube` on 2011-07-30 13:39:18

<hr>
 * *Attachment: [fc-refetch-source.diff](https://storage.googleapis.com/google-code-attachments/fullcalendar/issue-1061/comment-0/fc-refetch-source.diff)*

Imported with **2** stars.

**Comment by mchapman** [comment]
+1

**Comment by pavankumarkatakam** [comment]
+1

**Comment by caseyjhol** [comment]
I added this in PR #3103.

**Comment by arshaw** [comment]
[Code complete](https://github.com/fullcalendar/fullcalendar/commits/v2.8.0-dev) and slated for v2.8.0 ([wip docs](https://github.com/fullcalendar/fullcalendar/commits/docs-2.0)). Will notify when released.

API: pass in a single argument to the new `refetchEventSources` method

**Comment by arshaw** [comment]
Released in [v2.8.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.8.0).

http://fullcalendar.io/docs/event_data/refetchEventSources/
(called with a single source argument)
